base/core/src/main/java/org/openscience/cdk/config/IsotopeFactory.java

```diff
@@ -215,7 +215,30 @@ public IIsotope getMajorIsotope(int elem) {
             if (major != null)
                 this.majorIsotope[elem] = major;
             else
-                logger.error("Could not find major isotope for: ", elem);
+                switch(elem) {
```

Comment:
Not OK need to be in the BODR and then compiled in @egonw can chime in. See slide 4 https://www.slideshare.net/NextMoveSoftware/building-on-sand-standard-inchis-on-nonstandard-molfiles on issues with InChI isotopes, e.g. Sg.